pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -843,7 +843,9 @@ func ensureSystemCgroups(rootCgroupPath string, manager *fs.Manager) error {
 
 		allPids, err := cmutil.GetPids(rootCgroupPath)
 		if err != nil {
-			errs = append(errs, fmt.Errorf("failed to list PIDs for root: %v", err))
+			if attemptsRemaining < 0 {
+				errs = append(errs, fmt.Errorf("failed to list PIDs for root: %v", err))
+			}
 			continue
```

Comment:
IMO, that's wrong
`errs = []error{}` in line#841 has the same function

Reply:
If attemptsRemaining >= 0, the loop would continue, leading to code on line 841.

See also Matt's comment.